chrome-launcher/chrome-launcher.ts

```diff
@@ -253,7 +253,7 @@ export class Launcher {
           if (isWindows) {
             execSync(`taskkill /pid ${this.chrome.pid} /T /F`);
           } else {
-            process.kill(-this.chrome.pid);
```

Comment:
Just told Paul, but it was to kill the entire process group. I can't remember where, but in some cases (on some platform?) Chrome was not the top level process and wasn't being killed without it, and it works equivalently everywhere else. Probably should have documented it :)